packages/angular/projects/ui-angular/package.json

```diff
@@ -9,6 +9,11 @@
     "generate:css": "npx tailwindcss -i ./styles/component.tailwind.css -o ./styles/component.css"
   },
   "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws-amplify/amplify-ui",
+    "directory": "packages/angular/projects/ui-angular"
```

Comment:
@wlee221 Can you confirm this is the directly we should point to?

Reply:
Yep! Angular weirdness 😛